Content.Server/Worldgen/Systems/Debris/DebrisFeaturePlacerSystem.cs

```diff
@@ -162,6 +163,12 @@ private void OnDebrisShutdown(EntityUid uid, OwnedDebrisComponent component, Com
         var failures = 0; // Avoid severe log spam.
         foreach (var point in points)
         {
+            if (component.OwnedDebris.TryGetValue(point, out var existing))
+            {
+                DebugTools.Assert(Exists(existing));
+                continue;
+            }
+
```

Comment:
I'm not actually sure if this is the right fix, or what exactly this system is meant to do when entities already exist. @moonheart08